docs/product-user-guide/src/main/asciidoc/building-custom-dashboard-widgets-creating-dashboard-proc.adoc

```diff
@@ -0,0 +1,9 @@
+[id='building-custom-dashboard-widgets-creating-dashboard-proc']
+= Creating a custom dashboard widget
+
+There are four stages in the creation of a custom dashboard widget:
+
+* Data sets authoring - In this stage, you define a data set for accessing the data and displaying it through the pages. For more information, see xref:data_sets_add_proc[Adding data sets].
+* Pages authoring - In this stage, you create the pages which are used to display the data from the data sets, among other things. For more information, see xref:_building_custom_dashboard_widgets_creating_pages_proc[Creating pages].
+* Publication - In this stage, navigation between pages are defined when you create the custom navigation trees or modify the existing default one (*Workbench* tree). For more information, see xref:building-custom-dashboard-widgets-creating-navigation-tree-proc[Creating a navigation tree] or xref:building-custom-dashboard-widgets-editing-navigation-tree-con[Editing a navigation tree].
+* Security management - In this stage, role and group permissions are set which defines the privileges a user enjoys when working on {PRODUCT}. For more information, see xref:business-central-settings-security-proc[Security settings].
```

Comment:
Add the following note:

"NOTE: The set of stages listed above is the procedure to follow for users coming from older versions who want to migrate their dashboards to the new tooling. Notice, there isn't an automatic migration path or tooling to assist in that regards. Users must recreate their dashboards from scratch.

Reply:
Added note as suggested. 
Rephrased the note to conform to RH guidelines